index.html

```diff
@@ -208,5 +208,75 @@ <h6>About me</h6>
         </div>
       </section>
     </div>
+    <section class="contact">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/xyyaqdyk" method="POST">
+        <ul>
+          <li>
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="Email address"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              id="msg"
+              name="user_message"
+              placeholder="write me something..."
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
+          <li class="btn-center">
+            <button type="submit">Get in touch</button>
+          </li>
+        </ul>
+      </form>
+    </section>
+    <footer>
+      <ul>
+        <li>
+          <a href="https://github.com/Tupai566"
+            ><img src="./Images/GitHub2.png" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/tupai-mazumder-16109a243/"
+            ><img src="./Images/Linkedin2.png" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.facebook.com/tupai.mazumder/"
+            ><img src="./Images/V-Hand2.png" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/imtupai566"
+            ><img src="./Images/Twitter2.png" alt=""
```

Comment:
- [ ] Please kindly ensure that images have the alt image text 👍 . This is because alt attribute messages enable screen readers to read the information about on-page images for the benefit of a person with complete lack of sight, visually impaired, or who is otherwise unable to view the images on the page.

Reply:
Thanks for the detailed description, resolved them.